onboarding.adoc

```diff
@@ -46,7 +45,33 @@ No two open source communities are the same.
 Even for seasoned open source contributors, when they join a new community they will need to get familiar with new tools, new processes, and even new terminologies.
 Depending on the community, seemingly simple terms like contributions, community members, projects, upstream, etc.
 may mean something slightly different and can easily cause confusion for new community members.
-  
+
+== Onboarding basics
+
+As you work to make your project's onboarding process and experience most effective, ask yourself these questions:
+
+- What can contributors add to the project?
+- Where do contributors add to the project?
+- How do contributors add to the project?
+
+Answering the first question about your project's onboarding process requires a good deal of work work.
+Incoming contributors need to know what would be the most helpful contributions they could make.
+Quite a few projects have set up tagging on their source code repositories for "first-time contributors" or something similar, which is an excellent way to direct incoming contributors to solving issues and bugs while also introducing newcomers to the source material of the project.
+
+The second question—regarding "where to contribute"—is one that projects don't pose as often as you'd think.
+Project veterans might dismissively say, "what you need is on Git-something-or-other."
+But in which specific repository are the material available?
+Under what organization?
+And what about other, similarly named projects?
+Ensure your project website features clear links to places where new contributors can make an impact on the project. These links shouldn't be buried, either; they should be right on the home page, if possible.
+
+Third is teh question of "how to contribute."
```

Comment:
```suggestion
Third is the question of "how to contribute."
```

Reply:
Gah! Silly mistake. Thanks for catching, @rpaik.